diesel/src/backend.rs

```diff
@@ -6,6 +6,18 @@ use query_builder::bind_collector::BindCollector;
 use query_builder::QueryBuilder;
 use sql_types::{self, HasSqlType};
 
+use std::marker::PhantomData;
+
+pub trait FamilyLt<'a> {
```

Comment:
The trait and the type below should be `#[doc(hidden)]` to remove them from our public api.